v4l2-request-test.c

```diff
@@ -64,19 +64,20 @@ struct format_description formats[] = {
 
 static void print_help(void)
 {
-	printf("Usage: v4l2-request-test [OPTIONS] [SLICES PATH]\n\n"
+	printf("Usage: v4l2-request-test [OPTIONS]\n\n"
 	       "Options:\n"
-	       " -v [video path]                path for the video node\n"
-	       " -m [media path]                path for the media node\n"
-	       " -d [DRM path]                  path for the DRM node\n"
-	       " -D [DRM driver]                DRM driver to use\n"
-	       " -s [slices filename format]    format for filenames in the slices path\n"
-	       " -f [fps]                       number of frames to display per second\n"
-	       " -P [video preset]              video preset to use\n"
-	       " -i                             enable interactive mode\n"
-	       " -l                             loop preset frames\n"
-	       " -q                             enable quiet mode\n"
-	       " -h                             help\n\n"
+	       " -v | --video_path    [video path]       path for the video node\n"
+	       " -m | --media_path    [media path]       path for the media node\n"
+	       " -d | --drm_path      [DRM path]         path for the DRM node\n"
+	       " -D | --drm_driver    [DRM driver]       DRM driver to use\n"
+	       " -s | --slices_path   [slices path]      path to stored video slices\n"
+	       " -S | --slices_format [slices format]    format for filenames in the slices path\n"
```

Comment:
You are making -s the slices path and -S the slides format while the latter was -s before and it looks like the opt switch/case is not updated for that.

Reply:
Sorry. My flegmatism. Will be consistent. All path references are lower case letters. So -s will represent the slices_path and -S  the slices_format_filename. WIP